### DISCUSSION TITLE
Show Navigation Panel bug on Android

### DISCUSSION THREAD

**Discussion by Shattapp** [standardnotes/forum#2918] [Suggestions]
**Describe the bug**
If you toggle on Show Navigation Panel on an Android device, in my case the Samsung Galaxy Tab S8 Ultra, and then click on a note, the navigation panel is gone. And the "toggle on" is now off instead. 

I noticed this the first time in the end of November/beginning of December.

**To Reproduce**
1. Toggle on Show Navigation Panel
2. Click on a note
3. Navigation panel is gone

**Expected behavior**
That the Navigtion Panel should stay being shown.

**Smartphone (please complete the following information):**
 - Device: Galaxy Tab S8 Ultra, Tab S7 Plus
 - OS: Android 13
 - Browser: Edge
 - Version 3.125.0

**Comment by effieeee** [comment]
Hi @Shattapp, I wanted to clarify if you're using the **web app** through the browser on your device, and not the Android app? The toggle for that panel is not usually needed on the Android app since that panel can be toggled by the **top left hamburger icon**

![image](https://user-images.githubusercontent.com/54989748/206729635-c22cf54a-cf3f-4fb9-86ae-1099eb9e176b.png)

**Comment by Shattapp** [comment]
Hi,

No, I'm using the android app and not the web app :-) And yes, this more of a "bug" then a bug.

And sure, the show navigation panel isn't that necessary... 

It's more that this worked without any issues a few versions ago

and 

1 - you have the option in the menu but it doesn't do anything
2- I kind of like to have all 3 columns :-) I use the Tab S8 Ultra with a 14,6-inch screen so there is space to play with... It works on the desktop version and on the webapp version.

br
Mattias

**Comment by effieeee** [comment]
I see, we'll take note of your preference especially since your tablet has quite a large screen. 👍
In the meantime though, maybe it could be a workaround if you save the web app (app.standardnotes.com) to your device's home screen instead? You'll at least be able to see all 3 panels open through there

**Comment by Shattapp** [comment]
👍🙂
